gitlab/visualcompare.sh

```diff
@@ -64,8 +64,16 @@ do
         -X POST \
         -H "Authorization: token $GH_BOT_TOKEN_OBSCURED" \
         -H "Accept: application/vnd.github.v3+json" \
-        -d "{\"state\": \"$STATE\", \"target_url\": \"$CI_JOB_URL/artifacts/file/$FILE\", \"description\":\"UI changes\", \"context\": \"UI diffs ($dir)\"}"
-      exit 1
+        -d "{\"state\": \"$STATE\", \"target_url\": \"$CI_JOB_URL/artifacts/$FILE\", \"description\":\"UI changes\", \"context\": \"UI diffs ($dir)\"}"
+      if [ $STATE=="failure" ]; then
+        cat <<EOF > $failingchanges/README
+List you changed endpoints in a message on the PR. Regex is allowed.
```

Comment:
This sentence feels odd?

Reply:
I think because it is, changed it a bit and made the syntax also slightly better